src/main/java/domain/Player.java

```diff
@@ -30,18 +32,34 @@ private void validatePlayerName(String playerName) {
     }
 
     private void validateWord(String playerName) {
-        if (!PLAYER_NAME_PATTERN.matcher(playerName).matches()) {
+        if (isNotMatches(playerName)) {
```

Comment:
```suggestion
        if (!isMatches(playerName)) {
```
여기도 한번만 !를 쓰도록 해보면 좋을것 같아요

Reply:
저 같은 경우는 오히려 메서드의 내부 로직을 보지 않고, 네이밍만 보더라도 이 메서드는 일치하지 않을 때 true 를 반환하겠구나! 라는 느낌을 주는게 더 가독성이 좋아보이는데요.

영이는 이 부분에 대해서 어떻게 생각하시나요?

Reply:
여기는 isNotMatches가 하나의 조건문으로 처리되어서 괜찮은것 같습니다
하지만 Command의 isOutOfRange는 내부 로직에서 조건문을 && 으로 두가지 경우를 체크하는데 이를 !하는것은 다른 사람이 조건을 수정할때 여러번 생각해봐야하는 번거로움이 있을것 같네요